libs/platform/src/lib/components/action-bar/action-bar.component.ts

```diff
@@ -103,6 +104,7 @@ export class ActionBarComponent implements OnInit {
 
     enableEditTitle(editing: boolean) {
         this.editing = editing;
+        setTimeout(() => this.inputTitle.nativeElement.focus(), 0);
```

Comment:
There is a slight chance here that if the component is destroyed before the setTimeout finishes it will create a memory leak.

You should clear this timeout on destruction of the component. First create a reference to the timeout:
```
    enableEditTitle(editing: boolean) {
        this.editing = editing;
        this.timeout = setTimeout(() => this.inputTitle.nativeElement.focus(), 0);
        ...
```

Add `implements OnDestroy` to the component then add the following:
```
ngOnDestroy() {
    if (this.timeout) {
        clearTimeout(this.timeout);
    }
}
```